pkg/extractors/summarizer.go

```diff
@@ -152,16 +150,19 @@ func summarize(
 		promptTokens = 250
 	}
 
+	summarizerMaxOutputTokens := appState.Config.Extractors.Summarizer.MaxTokens
+
 	// We use this to determine how many tokens we can use for the incremental summarization
 	// loop. We add more messages to a summarization loop until we hit this.
-	summarizerMaxInputTokens := maxTokens - SummaryMaxOutputTokens - promptTokens
+	summarizerMaxInputTokens := maxTokens - summarizerMaxOutputTokens - promptTokens
```

Comment:
@danielchalef Is this what you were mentioning about needing to know the length of the summary? I updated that part here as well so it assumes the max output would be what you have configured (which defaults to 512 as you had before).